test/parallel/test-http-remove-connection-header-persists-connection.js

```diff
@@ -45,14 +45,20 @@ function makeHttp11Request(cb) {
 function makeHttp10Request(cb) {
   // We have to manually make HTTP/1.0 requests since Node does not allow sending them:
   const socket = net.connect({ port: server.address().port }, function() {
+    socket.on('error', common.mustNotCall());
+
     socket.write('GET / HTTP/1.0\r\n' +
                'Host: localhost:' + server.address().port + '\r\n' +
                 '\r\n');
-    socket.resume(); // Ignore the response itself
+    socket.end(); // we are done sending data
```

Comment:
```suggestion
```

This invalidate the test. The socket should be closed by the server.

Reply:
This only closes one end of the socket, as I understand it. But I will remove it.